sql-cli/sql_cli/run_dag.py

```diff
@@ -25,6 +26,11 @@
 log = logging.getLogger(__name__)
 log.setLevel(logging.INFO)
 
+# This was added to airflow.utils.session as part of Airflow 2.2.4, we're copying it here for backwwards
+# compatibility (currently we support Airflow 2.1 onwards)
+# It seems like a hack to avoid MyPy complaining we're assigning None to a variable that should be an Airflow session
+NEW_SESSION = airflow_settings.SASession = cast(airflow_settings.SASession, None)
```

Comment:
Can we not just use type: ignore instead? Seems iffy to make these kinds of hacks just for mypy

Reply:
@dimberman the `main` version of the SQL CLI is already using this same exact code, from the moment we relied on Airflow's `NEW_SESSION` during the implementation of the `run` command. The only difference is that for backwards compatibility, I brought this part of the code to our code base and realised it was a hack:
https://github.com/apache/airflow/blob/main/airflow/utils/session.py#L84

I agree this can be improved and I suggest it is done as part of a separate PR.